index.bs

```diff
@@ -2932,6 +2924,15 @@ nothrow>WritableStreamFinishPendingCloseWithError ( <var>stream</var>, <var>reas
   1. Perform ! WritableStreamRejectClosedPromiseIfAny(_stream_).
 </emu-alg>
 
+<h4 id="writable-stream-is-ready-promise-pending" aoid="WritableStreamIsReadyPromisePending"
+nothrow>WritableStreamIsReadyPromisePending ( <var>stream</var>, <var>state</var>, <var>wasAborted</var> )</h4>
+
+<emu-alg>
```

Comment:
I really don't like passing in state and wasAborted. Can we get those from the stream instead? It seems they are almost always computed the same way, so that would just be more good deduplication.